pkg/constants/constants.go

```diff
@@ -24,6 +24,11 @@ const (
 	// TODO(#23): Use the kubernetes.io domain when graduating APIs to beta.
 	QueueAnnotation = "kueue.x-k8s.io/queue-name"
 
+	// ParentWorkloadAnnotation is the annotation used to mark child jobs to be
+	// controller by Kueue and ignored from scheduling. It's value is the name
+	// of the workload controller by the main job.
+	ParentWorkloadAnnotation = "kueue.x-k8s.io/parent-workload"
```

Comment:
But what's the relationship with kubernetes batch Job? MPI job is migrating to use batch Job instead?

Reply:
Yes, v2 creates the Launcher via batch/v1 Job.

Reply:
https://github.com/kubeflow/mpi-operator/blob/c21942d1e27d196d06078f67f39a6f7a0e990dee/pkg/controller/mpi_job_controller.go#L1390-L1413

Reply:
Can you validate in the webhook that the value is a valid CRD name?
Maybe we should also validate that it's immutable?

Reply:
> Can you validate in the webhook that the value is a valid CRD name?
Maybe we should also validate that it's immutable?

I agree.

Reply:
Job webhook extended, including the check to ensure it is immutable.